package/tests/snapshots/integration__package__tests__integration__inputs__path__conflicting-index-deps__Nickel-pkg.ncl.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: package/tests/integration.rs
+expression: e.to_string()
+---
+Because 'Path <filtered> depends on github:example/leaf 0.1.0 and top-level package 0.1.0 depends on 'Path <filtered> top-level package 0.1.0 depends on github:example/leaf 0.1.0.
+And because top-level package 0.1.0 depends on github:example/leaf 0.1.1, top-level package 0.1.0 is forbidden.
```

Comment:
Is there more to the error text that the users see? I feel like it's missing a main label or something

Reply:
Yes, the cli wraps this error in a `report_with_msg`, so it gets a main label. Before you wrote this comment, the main label was uselessly generic ("packaging error"), but I just updated it so that for this resolution failure it will be "failed to resolve package versions".